e2e/basics.spec.js

```diff
@@ -146,6 +153,23 @@ describe('Basic React Native and Interaction Support', () => {
       assert(res.length).equal(1);
       assert(res[0]['i']).equal('foobar');
     });
+
+    it('should reset identity', async () => {
+      const { err1, err2, res1, res2 } = await new Promise(resolve => {
+        testUtil.findEventInRedisRequests({ t:  'pressInTestEvent2'}, (err1, res1) => {
+          testUtil.findEventInRedisRequests({ t:  'BASICS_SENTINEL'}, (err2, res2) => {
+            resolve({ err1, err2, res1, res2 });
+          });
+        });
+      });
+
+      assert.not.exist(err1);
+      assert.not.exist(err2);
+      assert(res1.length).not.equal(0);
+      assert(res2.length).not.equal(0);
+      assert.not.exist(res2[0]['i']);
```

Comment:
Would you mind adding a comment as to what these keys are, and what you're actually checking here?